shell/assets/translations/en-us.yaml

```diff
@@ -224,6 +224,7 @@ nav:
   header:
     setLoginPage: Set as login page
     restoreCards: Restore hidden cards
+    showHideBanner: Show/Hide Banner
```

Comment:
nit: Would it make sense for this to read "Toggle Banner"?

Reply:
I think Show/hide is clear what it does - I guess it would be 'Toggle banner on/off' or 'Toggle banner visibility'